### PR TITLE
MOSIP-15746 : Updated error code for UISpec repeated upload

### PR DESCRIPTION
**Pull request by balaji-alluru**

### CODE REVIEW

File: admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UISpecServiceImpl.java [hunk 1]
```diff
@@ -149,14 +155,19 @@ public UISpecResponseDto defineUISpec(UISpecDto dto) {
 		uiSpecEntity.setId(UUID.randomUUID().toString());
 		uiSpecEntity.setIsDeleted(false);
 		uiSpecEntity.setEffectiveFrom(LocalDateTime.now(ZoneId.of(ZoneOffset.UTC.getId())));
+		List<UISpec> uispeclist = uiSpecRepository.findUISpecbyDomainandType(dto.getDomain(), dto.getType());
```
Comment (ase-101) [inline admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UISpecServiceImpl.java:158]: Follow method naming convention findUISpecByDomainAndType

File: admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UISpecServiceImpl.java [hunk 2]
```diff
@@ -12,12 +12,17 @@
...
+import org.hibernate.JDBCException;
+import org.hibernate.exception.ConstraintViolationException;
+import org.hibernate.exception.JDBCConnectionException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.cache.annotation.CacheEvict;
 import org.springframework.cache.annotation.Cacheable;
 import org.springframework.dao.DataAccessException;
+import org.springframework.dao.DataIntegrityViolationException;
```
Comment (ase-101) [inline admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UISpecServiceImpl.java:25]: Remove unused imports

Comment (balaji-alluru) [reply admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UISpecServiceImpl.java]: Removed unused Imports

File: admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UISpecServiceImpl.java (hunk 1 above)
Comment (balaji-alluru) [reply admin/kernel-masterdata-service/src/main/java/io/mosip/kernel/masterdata/service/impl/UISpecServiceImpl.java]: Updated the name as per your comment

**Review by ase-101 — APPROVED** [review]

**Pull request merged**